windows/src/global/delphi/packages/Keyman.System.PackageInfoRefreshKeyboards.pas

```diff
@@ -185,7 +185,7 @@ function TPackageInfoRefreshKeyboards.CheckKeyboardLanguages: Boolean;
     begin
       with TBCP47Tag.Create(lang.ID) do
       try
-        if not IsValid(msg) then
```

Comment:
In windows/src/developer/TIKE/compile/ValidateKeyboardInfo.pas, the value `False` was inserted instead of the `True` seen here.  Why the difference?

Reply:
Good question. In .keyboard_info files, we allow codes like `und-fonipa`. However, in package files, we need to be more stringent because codes like that are not supported in Windows (and some other OS as well AFAIK). So `True` is passed when we want to be more persnickety.